src/main/java/com/smartcar/sdk/data/ApplicationPermissions.java

```diff
@@ -53,4 +55,12 @@ public void setPaging(ResponsePaging paging) {
   public void setPermissions(String[] permissions) {
     this.permissions = permissions;
   }
+
+  @Override
+  public String toString() {
+    return "ApplicationPermissions{" +
```

Comment:
Suggestion: Use `this.getClass().getName()` instead. That way if we ever rename the class, or copy/pasta it for a similar POJO, we don't have to remember to update this method.

This applies to all the `toString()` implementations that contain the class name, but I'll just leave this one comment.